README.md

```diff
@@ -94,10 +94,10 @@ bugsnag {
 }
 ```
 
-If you disable automatic uploading, you can still run the upload task manually, with the uploadBugsnag*Variant*Mapping task:
+If you disable automatic uploading, you can still run the upload task manually, with the uploadBugsnag*VariantOutput*Mapping task:
 
 ```shell
-./gradlew clean assembleRelease packageRelease ${moduleName}:uploadBugsnag${variantName}Mapping
+./gradlew clean build uploadBugsnag${variantOutputName}Mapping
 ```
 
 In this example, $moduleName is the module being built (e.g. 'app'), and $variantName is the build variant name (e.g. 'appProRelease')
```

Comment:
Might be good to show a splits example here.

Reply:
I've added in a splits example.